dear_petition/common/models.py

```diff
@@ -0,0 +1,55 @@
+from itertools import chain
```

Comment:
Is there a better location for code that is used by models in petition, sendgrid, and users?